css/styles.css

```diff
@@ -98,18 +98,18 @@ body {
 .btn {
   text-decoration: none;
   display: inline-block;
-  border: 2px solid #c761a2; 
+  border: 1px solid #ff52bf; 
   padding: .3rem 1.6rem;
   font-size: 1.1rem;
-  position: absolute;
-  -webkit-transition: all 0.8s;
+  position: relative;
+  -webkit-transition: all 0.3s;
   transition: all 0.3s;
```

Comment:
The transition should be lesser so that it can render faster on any screen size.